support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -67,12 +89,17 @@ const router = () => {
 					{countryIds.map((countryId) => (
 						<>
 							<Route
-								path={`/${countryId}/contribute/`}
-								element={firstStepLandingPage}
-							/>
-							<Route
-								path={`/${countryId}/contribute/:campaignCode`}
-								element={firstStepLandingPage}
+								path={`/${countryId}/contribute/:campaignCode?`}
+								element={
+									isThreeTierAndOneOffUrlParam() ? (
```

Comment:
I know this is all part of the test, but a comment with the context the PR gives that we're doing this to avoid changing the `www` banners and go straight to checkout might help later.